Source/WebKit/WebProcess/Network/webrtc/LibWebRTCSocket.h

```diff
@@ -58,8 +49,9 @@ namespace WebKit {
 
 class LibWebRTCSocketFactory;
 
-class LibWebRTCSocket final : public rtc::AsyncPacketSocket, public CanMakeWeakPtr<LibWebRTCSocket>, public Identified<WebCore::LibWebRTCSocketIdentifier> {
+class LibWebRTCSocket final : public CanMakeCheckedPtr<LibWebRTCSocket>, public rtc::AsyncPacketSocket, public Identified<WebCore::LibWebRTCSocketIdentifier> {
```

Comment:
We should leave `rtc::AsyncPacketSocket` as *first* base class, it is much safer. Otherwise, you could end up introducing crashes if existing code relies on reinterpret_cast<>() or conversion to and from `void*`.

Reply:
If you're curious this may shade some light: https://www.sjbrown.co.uk/posts/always-use-static-cast/